kadai-4/src/pages/dashboard.vue

```diff
@@ -90,16 +113,35 @@
       wallet() {
         return this.$store.getters.wallet
       },
+      userList(){
+        return this.$store.getters.userList
+      }
     },
     methods: {
-      openModal(userName,wallet){
+      openUserInfo(userName,wallet){
         this.showContent = true
         this.displayUserName = userName
         this.displayWalletData = wallet
       },
+      throwWallet(sendUid,sendUserWallet){
+        this.throwWalletContent = true
+        this.sendUserUid = sendUid
+        this.sendUserWallet = sendUserWallet
+      },
       closeModal(){
         this.showContent = false
       },
+      closeWalletModal(){
+        this.throwWalletContent = false
+      },
+      sendWallet(wallet){
+        const payload = {
+          wallet:wallet,
```

Comment:
オブジェクトは、keyとvalueが一致していれば以下のように省略できます。
```suggestion
          wallet,
```
これを意識しながら変数名を決めていくのも、可読性が高くなる方法の一つです。